src/objective-c/tests/PluginTest/plugin_option_test.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
would be nice to have a final test step that would take the generated objc stub and run a  bazel build to verify. Perhaps adding another TODO here to note that? thanks

Reply:
sure, but I think the priority should be to migrate as many tests as test we can to bazel BEFORE we start adding new coverage to the existing tests. It's better to have the same test coverage as before with faster and more reliable way of running them (bazel seems to give us that) then having some tests "perfect" while other tests are stuck with a messy and fragile setup.